awslabs/solution-spark-on-aws
# fix: launch environment should return HTTP 400 error when parameters are invalid

Issue #, if available:

Description of changes:
* Launch environment should return HTTP 400 error when parameters are invalid
* Added `jsonschema` library so we can define what our request body should look like and validate against it
* Added integration tests 

**Testing**
* Integration tests passed
* Manual testing passed

Checklist:




By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
